project/src/main/java/com/google/sps/data/ServletsLibrary.java

```diff
@@ -0,0 +1,168 @@
+package com.google.sps.data;
+
+import com.google.api.gax.longrunning.OperationFuture;
+import com.google.cloud.vision.v1.*;
+
+import java.util.*;
+ 
+import java.io.IOException;
+
+public class ServletsLibrary{
```

Comment:
Since these functions are really important to the project, you need to have unit tests so that you can confirm that they work.